app/importers/unpaywall_publication_importer.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+class UnpaywallPublicationImporter
+  def import_all
+    unless Rails.env.test?
+      pbar = ProgressBar.create(title: 'Importing publication data from Unpaywall',
```

Comment:
I've found that running ProgressBar in test is fine, so go ahead and nix the `Rails.env.test?` checks.